Tools/autotest/arducopter.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, binary,
         self.log_name = "ArduCopter"
 
         self.sitl = None
+        self.NOT_ARMABLE_MODES_LIST = ["AUTO", "AUTOTUNE", "BRAKE", "CIRCLE", "FLIP", "LAND", "RTL", "SMART_RTL", "AVOID_ADSB"]  # miss FOLLOW as need parameter change
+        self.NOT_SETTABLE_MODES_LIST = ["FLIP", "AUTOTUNE"]
```

Comment:
Why are these not settable? Do you mean, not settable while disarmed?

Reply:
yep, you cannot set those mode on ground

Reply:
I think the naming should clearer then, it really confused me (I had to go look why these weren't settable). Maybe NOT_DISARMED_SETTABLE_MODES_LIST? It's long, but it's clear.